### PR TITLE
Add attrchange@2.0.1

### PR DESCRIPTION
**Pull request by Piicksarn**
**git repo url:** https://github.com/meetselva/attrchange/tree/2.0.1
**Watch 15
  Star 132
 Fork 38**
@maruilian11  could you help me check out the PR for #6241 ? thank you~
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity 
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

**Comment by maruilian11** [comment]
@Piicksarn by **In the last release, The plugin code is refactored to move the extensions code to a new file attrchange_ext.js. The core plugin attrchange.js should still work independently.**
i think we should just host `attrchange.js`
and you seem to forgot requiredFiles field and repository field

**Comment by Piicksarn** [comment]
@maruilian11, 
i followed the usage example of attrchange to include the lib, and the link is shown below:
http://meetselva.github.io
And i changed the autoupdate from git to nom.
would you mind to checkout the pr again?
Thank you!

**Comment by maruilian11** [comment]
@PeterDaveHello could you please check this PR? thanks.

File: ajax/libs/attrchange/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  ],
+  "author": {
+    "name": "Selvakumar Arumugam",
+    "url": "http://meetselva.github.com/attrchange/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/meetselva/attrchange.git"
+  },
+  "licens": "MIT",
```
Comment (PeterDaveHello) [inline ajax/libs/attrchange/package.json]: licens?

**Comment by PeterDaveHello** [comment]
No minified `attrchange_ext.js`?

**Comment by Piicksarn** [comment]
@maruilian11, @PeterDaveHello, i have it updated!

File: ajax/libs/attrchange/package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    {
+      "basePath": "js",
+      "files": [
+        "attrchange*.js"
+      ]
+    }
+  ],
+  "requiredFiles": [
+    "attrchange_ext.min.js"
+  ]
```
Comment (PeterDaveHello) [inline ajax/libs/attrchange/package.json]: no more `requiredFiles`, see : https://github.com/cdnjs/cdnjs/issues/7264

**Comment by Piicksarn** [comment]
@maruilian11, i have removed the requiredFiles part!
pls help me for check~
thank u~

**Pull request merged**